src/test/ssmDocument/commands/openDocumentItem.test.ts

```diff
@@ -62,20 +57,23 @@ describe('openDocumentItem', async function () {
     }
 
     it('create DocumentItemNode and openDocumentItem functionality', async function () {
-        sandbox.stub(vscode.window, 'showSaveDialog').returns(Promise.resolve(vscode.Uri.file('test')))
-        sandbox.stub(picker, 'promptUser').onFirstCall().returns(Promise.resolve(fakeFormatSelection))
-        sandbox.stub(picker, 'verifySinglePickerOutput').onFirstCall().returns(fakeFormatSelectionResult)
+        sinon.stub(vscode.window, 'showSaveDialog').returns(Promise.resolve(vscode.Uri.file('test')))
+        sinon.stub(picker, 'promptUser').onFirstCall().returns(Promise.resolve(fakeFormatSelection))
+        sinon.stub(picker, 'verifySinglePickerOutput').onFirstCall().returns(fakeFormatSelectionResult)
 
         const documentNode = generateDocumentItemNode()
-        const openTextDocumentStub = sandbox.stub(vscode.workspace, 'openTextDocument')
-        await openDocumentItem(documentNode, fakeAwsContext)
+        const openTextDocumentStub = sinon.stub(vscode.workspace, 'openTextDocument')
+        await openDocumentItem(documentNode, fakeAwsContext, 'json')
         assert.strictEqual(openTextDocumentStub.getCall(0).args[0]?.content, rawContent.Content)
         assert.strictEqual(openTextDocumentStub.getCall(0).args[0]?.language, 'ssm-json')
     })
 
     function generateDocumentItemNode(): DocumentItemNode {
-        const ssmDocumentClient = new MockSsmDocumentClient()
-        sandbox.stub(ssmDocumentClient, 'getDocument').returns(Promise.resolve(rawContent))
+        const ssmDocumentClient = {
+            async getDocument() {
+                return rawContent
+            },
+        } as unknown as SsmDocumentClient
```

Comment:
This is a nice use of the escape-hatch that typescript/javascript gives us.

Reply:
The fact that types and code are decoupled is what makes Typescript so powerful.